doc/gov/chapters/chapter01.md

```diff
@@ -4,48 +4,34 @@
-
-* Consensus-Based Decision Making (Ch. 5)
-
-The approaches and processes CNTT contributors shall use to ensure fair representation for key decisions and release approvals. This includes mechanisms on how to manage dissension. 
-
-* Relationship to LFN Projects and GSMA (Ch. 6)
-
-Provides operational guidance for how CNTT operational structures shall interoperate with GSMA and LFN.
+The Anuket project operates under a Technical Steering Committee (TSC) and governing rules documented in the [Anuket Charter](https://anuket.io/wp-content/uploads/sites/119/2021/07/Anuket-Charter_073021.pdf) and in the Anuket Project Operations and Guidelines document. 
 
+<a name="1.2"></a>
+## 1.2 How to participate
+
+Participating in Anuket does not need any formal membership, and nothing to sign except the [CLA](https://anuket.io/cla/). Participation is open to anyone, whether you are an employee of an LFN member company, supporter, or observer. By participating, you automatically accept the individual anti-trust policies of LFN and GSMA, the joint Terms of Reference of LFN and GSMA the LFN Code of Conduct, as well as the LFN Trademark policy. 
+
+- [LFN Anti-Trust Policy](https://lfnetworking.org/Antitrust%20Slide.pdf) 
+- [GSMA Anti-Trust Policy](https://github.com/cntt-n/CNTT/blob/master/Anti-Trust_Notice_GSMA_20190911.png) 
+- [CNTT Terms of Reference (ToR) LFN | GSMA](https://wiki.lfnetworking.org/display/LN/Terms+of+Reference?src=contextnavpagetreemode)
+  - Note: If you are not self-employed, then you must ensure that your employer also agrees to the ToR before participating.
+- [Code of Conduct LFN](https://lfprojects.org/policies/code-of-conduct/)
+- [Trademark Policy](https://lfprojects.org/policies/trademark-policy/)
+
+Recommended checklist for participating in the Anuket community:
+  - Review all legal policies, terms, and conditions
+  - Signup for an [LFN ID](https://sso.linuxfoundation.org/login/?state=g6Fo2SBEbDAzLVFFN18teXdLMWZlUlVmWlVrMFpwZElHaUFtTqN0aWTZIGxCNVo3NExGOVpxNzJGT25YRW5vSkxMTlAwTENaLVdSo2NpZNkgcFZ0QzZ2R2ZmMnFBd0lxOXZVWGlhdTZzN3h6OWR3UUI&client=pVtC6vGff2qAwIq9vUXiau6s7xz9dwQB&protocol=oauth2&audience=https:%2F%2Fapi-gw.platform.linuxfoundation.org%2F&scope=openid%20profile%20email&response_type=code&response_mode=query&nonce=VldNTTJZdzc1UX5HVzBFMFd0dXBzTlc1UXpyQnJpVFlzfnY5WHA3OTl6SQ%3D%3D&redirect_uri=https:%2F%2Fmyprofile.lfx.linuxfoundation.org&code_challenge=VzQLzce3RLtJjTo1QmtKvZ3hrYd6J5U1yyQ4FXVodZ4&code_challenge_method=S256&auth0Client=eyJuYW1lIjoiYXV0aDAtc3BhLWpzIiwidmVyc2lvbiI6IjEuMTMuMiJ9)
+  - Signup for a [GitHub ID](https://github.com/signup?ref_cta=Sign+up&ref_loc=header+logged+out&ref_page=%2F&source=header-home)
+  - Signup for a [GitLab ID](https://gitlab.com/users/sign_up)
+  - [Find a sub-project](https://wiki.anuket.io/display/HOME/Projects)
```

Comment:
Can we be a bit more descriptive such as "Choose sub-projects and begin contributing; review the sub-project information and on how to participate including meetings" or ...

Reply:
Added in b6a13e7